physicalplan/unary/unary.go

```diff
@@ -27,16 +30,22 @@ type unaryNegation struct {
 func NewUnaryNegation(
 	next model.VectorOperator,
 	stepsBatch int,
+	op parser.ItemType,
 ) (model.VectorOperator, error) {
 	u := &unaryNegation{
 		next:       next,
 		stepsBatch: stepsBatch,
+		op:         op,
```

Comment:
Umm why do we need OP? It is unary negation so op is `-`. For `+` we don't have an additional operator to handle that, same does promql.

Reply:
You're right! Will remove the `op`.